src/promptflow-core/promptflow/_utils/flow_utils.py

```diff
@@ -81,17 +83,38 @@ def resolve_flow_path(
     else:
         flow_path = Path(flow_path)
 
-    if new:
-        if flow_path.is_dir():
-            return flow_path, DEFAULT_FLOW_YAML_FILE_NAME
-        return flow_path.parent, flow_path.name
+    if flow_path.is_dir():
+        target_folder = flow_path
+        dag_file_exist = (target_folder / FLOW_DAG_YAML).exists()
+        flex_file_exist = (target_folder / FLOW_FLEX_YAML).exists()
+        target_file = FLOW_FLEX_YAML if flex_file_exist else FLOW_DAG_YAML
+        if dag_file_exist and flex_file_exist:
+            raise ValidationException(
+                f"Both exist {FLOW_DAG_YAML} and {FLOW_FLEX_YAML} in the flow path {flow_path}, "
```

Comment:
Both {FLOW_DAG_YAML} and {FLOW_FLEX_YAML} exist in {flow_path}. Please specify a file or remove the extra YAML.

Reply:
done